cmd/soroban-cli/src/commands/network/container/start.rs

```diff
@@ -56,7 +62,7 @@ impl Cmd {
 }
 
 async fn run_docker_command(cmd: &Cmd) -> Result<(), Error> {
-    let docker = connect_to_docker(&cmd.docker_host).await?;
+    let docker = connect_to_docker(&cmd.container_args.docker_host).await?;
```

Comment:
Perhaps this should be a method on `container_args`?

Reply:
I like that idea since it would clean up this line. But on the other hand, would that make Args responsible for too much? LIke it would be handling argument related things, and now also starting a docker connection 🤔

Reply:
This is common across the crate. Personally, the struct that has the data needed should handle the logic. This way you can share the logic and args across commands.

Reply:
Cool, that makes sense. I'll get that change pushed up shortly